src/app.js

```diff
@@ -1 +1,38 @@
 'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+function copyFile() {
+  const terminalParameters = process.argv.slice(2);
+  const copyFromFile = terminalParameters[0];
+  const copyToFile = terminalParameters[1];
+
+  if (!copyFromFile || !copyToFile) {
+    global.console.log(
+      'You need to indicate which file you want to make a copy from',
+      'For example: file.txt file-copy.txt'
+    );
+  }
+
+  const pathToCopyFromFile = path.join(__dirname, copyFromFile);
```

Comment:
No need to join your paths with `__dirname`, just use it as given by user. Because of this, you can only copy files inside the `src/` folder. For example, currently this will not work (running from the project root):
```
node src/app.js readme.md readme2.md
```